tests/model_selection/test_hyperband.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+import scipy.stats as stats
+import math
+
+from distributed import Client
+from distributed.utils_test import loop, cluster
+
+from dask_ml.datasets import make_classification
+from dask_ml.model_selection import HyperbandCV
+
+
+class ConstantFunction:
+    def _fn(self):
+        return self.value
+
+    def get_params(self, deep=None, **kwargs):
```

Comment:
I think you could inherit from `sklearn.base.BaseEstimator` and remove `get_params` and `set_params`.

Reply:
Yeah, that's totally possible. It is pretty lightweight, and has 5 fairly lightweight functions: `{get, set}_params`, `__{get, set}state__`, `__repr__` and `_get_param_names(self)`.